.about.yml

```diff
@@ -86,15 +135,15 @@ stack:
 #   url: URL for detailed information
 #   badge: URL for the status badge
 services:
-- 
```

Comment:
This is also covered in the `component.yaml`, so I'd rather not have to maintain it in two places if it can be avoided.